packages/clay-form/src/SelectBox.tsx

```diff
@@ -7,11 +7,13 @@ import ClayButton, {ClayButtonWithIcon} from '@clayui/button';
 import classNames from 'classnames';
 import React from 'react';
 
+export type TItem = {
```

Comment:
Interesting pattern, eliminates duplicate code, I like it. Why the name `TItem`, should I use this name in the future if I have a similar pattern?

Reply:
We have an eslint rule that enforces all `type` names begin with "T" and all `interface` names begin with "I". 

I would say follow this pattern when we have some sort of re-used type. Since this shape of item is used in quite a few places, it made sense to make a single type for it.